app/controllers/admin/chips_controller.rb

```diff
@@ -3,7 +3,23 @@ def index
     @chips = Chip.all
   end
 
+  def new
+    @chip = Chip.new
+  end
+
+  def create
+    @chip = Chip.new(chip_params)
+    if @chip.save
+      flash[:notice] = "Successfully created Chip"
+      redirect_to admin_chips_path
+    else
+      flash.now[:error] = @chip.errors.full_messages(', ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.